robot-core/src/main/java/org/obolibrary/robot/RelatedObjectsHelper.java

```diff
@@ -2344,4 +2344,26 @@ private static void spanGapsHelper(
       }
     }
   }
+
+  /**
+   * Removes all of the axiom annotations for the given annotation property.
+   *
+   * @param ontology OWLOntology to get axioms
+   * @param property Annotation property to remove related axiom annotations.
+   */
+  public static void removeAxiomAnnotations(OWLOntology ontology, OWLAnnotationProperty property) {
+    OWLOntologyManager manager = ontology.getOWLOntologyManager();
+    for (OWLAxiom axiom : ontology.getAxioms()) {
```

Comment:
We dont have to specify IMPORTS.INCLUDED or IMPORTS.EXCLUDED here?

Reply:
method moved to `OntologyHelper.java` and imports excluded